src/puppetlabs/pcp/protocol.clj

```diff
@@ -37,16 +44,18 @@
 
 (def InventoryRequest
   "Data schema for http://puppetlabs.com/inventory_request"
-  {:query [Uri]})
+  {:query [Uri]
+   (s/optional-key :subscribe) s/Bool})
 
 (def InventoryResponse
   "Data schema for http://puppetlabs.com/inventory_response"
-  {:uris [Uri]})
+  {:uris [Uri]
+   :version s/Int})
```

Comment:
This... would impact version 1 clients, but we don't allow any of them to send inventory requests so I suppose that's fine.